main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+    "encoding/csv"
+    "github.com/gin-gonic/gin"
+    "io"
+    "log"
+    "os"
+    "strconv"
+)
+
+const CSV_ERROR_MESSAGE = "There was an error when attempting to read the csv file. Please try again later."
+const FILE_PATH = "covid.csv"
+
+type Country struct {
+    Name   string  `json:"country_name"`
+    Cases  int     `json:"cases"`
+    Deaths int     `json:"deaths"`
+    Ratio  float32 `json:"death_ratio"`
+}
+
+func csvError(context *gin.Context, err error) {
+    log.Println(err)
+    context.JSON(500, gin.H{
```

Comment:
Avoid the use of magic numbers.

instead of `500`, use `http.StatusInternalServerError`